### ISSUE TITLE
More concise wording

### ISSUE THREAD

**Issue by chrisnorth** [chrisnorth/design-a-space-telescope#9]
Cut down the amount of text on main webpage to make clearer.

Add "More information about <X>" to link to the guide.

**Comment by slowe** [comment]
If you add this as e.g. `<div class="moreinfo"><a href=\"#guide_instruments\" class=\"guidelink\">More information about instruments</a></div>` it will get styled to the right.

**Comment by chrisnorth** [comment]
In "designer" part of en_advanced.json and en.json.
